sierra/src/main/java/org/httprpc/sierra/Picker.java

```diff
@@ -112,10 +114,28 @@ protected void processFocusEvent(FocusEvent event) {
 
         switch (event.getID()) {
             case FocusEvent.FOCUS_GAINED -> showPopup();
-            case FocusEvent.FOCUS_LOST -> hidePopup();
-        }
-    }
-
+            case FocusEvent.FOCUS_LOST -> {
+				// hide the popup, if focus did not move to it
```

Comment:
In general I try to avoid using comments when the code itself is self-explanatory (which in this case, I think it is).